app/assets/javascripts/hubstats/application.js

```diff
@@ -14,45 +14,83 @@
 //= require jquery_ujs
 //= require select2
 //= require hubstats/bootstrap
+//= require bootstrap-datepicker
 //= require_tree .
 
 $(document).ready( function() {
+  defaultDateRange()
   setDateRange()
 });
 
+function defaultDateRange() {
+  $('.input-daterange').datepicker({
+    "todayHighlight": true,
+    "endDate": "Today",
+    "todayBtn": true
+  });
+};
+
 function setDateRange() {
-  var index = readCookie("hubstats_index") || 2;
-  var timer = document.getElementById("time-select");
+  if ((readCookie("hubstats_index") === "null~~null") || (readCookie("hubstats_index") === null)) {
+    var index = getDefaultDateRange();
+  } else {
+    var index = readCookie("hubstats_index")
+  }
 
-  timer.selectedIndex = index;
+  var timer = document.getElementById("submitDateRange");
+  dates = index.split("~~")
+  $('.input-daterange').find('[name="start"]').datepicker('update', new Date(dates[0]));
+  $('.input-daterange').find('[name="end"]').datepicker('update', new Date(dates[1]));
 
-  timer.onchange = function() {
-    createCookie("hubstats_index",this.selectedIndex,1);
+  timer.onclick = function() {
```

Comment:
The `setDateRange` function is getting a bit big and this event handler arguably belongs better in the init function for the datepicker.  Consider moving the event handler to its own function or within the init function.